src/ActionsTests/Misc/ExtensionMethodsTests.cs

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved.
```

Comment:
>Copyright [](start = 3, length = 9)

This file was pulled in accidentally. I can pull it out if there's a desire for it